### PR TITLE
Fix: Broken links #1299

### PR DESCRIPTION
**Pull request by punwai**
@muarachmann Added this new PR instead. The links in the PHP files were broken #1299.

### CODE REVIEW

**Comment by punwai** [comment]
This affects 4 things: Administration -> Alert, Adiministration -> Rules and 2 other things.

**Comment by Trodrige** [comment]
This may not be related to the issue you set out to solve, but I get this when I try Admin->Rules

librehealth_ehr


> This affects 4 things: Administration -> Alert, Adiministration -> Rules and 2 other things.

what are the 2 other things?😁

**Comment by punwai** [comment]
@Trodrige I am not sure as to why that error happens, it seems to work fine for mine with a fresh fetch from the upstream branch and only edits to those files.
screen shot 2018-12-04 at 18 28 44

The other 2 errors I do not know how to get to from the interface but they are at:
http://localhost/lh-ehr/interface/super/rules/index.php?action=browse!plans_config
http://localhost/lh-ehr/interface/super/rules/index.php?action=edit!action

**Comment by punwai** [comment]
I've also found a MySQL error when trying to use one of the components. I will create a issue on that.
screen shot 2018-12-04 at 18 38 04

**Comment by muarachmann** [comment]
@Trodrige works ok for me. I just tested.....

screen shot 2018-12-05 at 3 35 46 am

I think on my end the broken links are fine. Looking at the codes though..  great @punwai 

screen shot 2018-12-05 at 3 37 38 am

File: interface/super/rules/controllers/browse/view/list.php [hunk 1]
```diff
@@ -6,8 +6,8 @@
  // as published by the Free Software Foundation; either version 2
  // of the License, or (at your option) any later version.
 
- require_once("../../../../../globals");
- require_once("$srcdir/headers.inc.php");
+ require_once(dirname(__FILE__)."/../../../../../globals.php");
+ require_once($GLOBALS['srcdir']."/headers.inc.php");
```
Comment (muarachmann) [inline interface/super/rules/controllers/browse/view/list.php]: @punwai i guess the global.php file was not required. thanks for fixing the link for that. Also i think it is not necessary to do `$GLOBALS['srcdir']` cause globals is already required 😃 . Try using the `$srcdir` and lets see what it gives

File: interface/super/rules/controllers/browse/view/plans_config.php [hunk 2]
```diff
@@ -24,8 +24,8 @@
  * @link http://librehealth.io
 */
 
- require_once("../../../../../globals");
- require_once("$srcdir/headers.inc.php");
+require_once(dirname(__FILE__)."/../../../../../globals.php");
+require_once($GLOBALS['srcdir']."/headers.inc.php");
 ?>
```
Comment (muarachmann) [inline interface/super/rules/controllers/browse/view/plans_config.php]: Same here. Just try that and get back to the PR to see what it gives

Comment (punwai) [reply interface/super/rules/controllers/browse/view/plans_config.php]: @muarachmann I've tried changing that line to 
 require_once("$srcdir/headers.inc.php");
but it seems to give an error.

**Comment by Trodrige** [comment]
I'll have to reinstall EHR on my end to test this. I still have the BaseController.php warning showing up.

**Comment by aethelwulffe** [comment]
NOTE:
The table changed for this.
"INSERT INTO `clinical_rules` (`id`, `pid`, `active_alert_flag`, `passive_alert_flag`, `patient_reminder_flag`, `release_version`, `web_reference`, `access_control`, `pqrs_code`, `pqrs_individual_2016_flag`, `pqrs_group_type`, `active`)
"cqm" and "amc" are outmoded regulatory format things that were dead years ago.
We can add the vars in, but the query only works when the table matches.
  Either edit it, or we drop the whole related module.  It was created to meet the requirements checklist of an EHR certification from about ten-twelve years ago, done poorly at the time, and is not longer relevant even if they did work without killing the database server.

**Pull request merged**